### ISSUE TITLE
Overlay from hyprland-protocols breaks lots of package in NixOS

### ISSUE THREAD

**Issue by L-Trump** [hyprwm/Hyprland#7930]
### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Regression?

Yes

### System Info and Version

Not important

### Description

The commit https://github.com/hyprwm/Hyprland/commit/14942bca60cc7d85e8238a151bd444112601ebe6 introduces the hyprland-protocols to the flake of Hyprland, including its overlay. However, the overlay in hyprland-protocols includes a `version` package. This breaks lots of nixos packages like `lxqt.libqtxdg`, since these packages have an argument called `version`. See below:

https://github.com/hyprwm/hyprland-protocols/blob/5a11232266bf1a1f5952d5b179c3f4b2facaaa84/flake.nix#L30C1-L36C7
```nix
    overlays = {
      hyprland-protocols = final: prev: {
        hyprland-protocols = final.callPackage ./nix/default.nix {inherit version;};
        version = version + "+date=" + (mkDate (self.lastModifiedDate or "19700101")) + "_" + (self.shortRev or "dirty");
      };
      default = self.overlays.hyprland-protocols;
    };
```

### How to reproduce

Build NixOS with Hyprland overlay and package `lxqt.lxqt-policykit`.

**Comment by vaxerski** [comment]
@fufexan

**Comment by fufexan** [comment]
Wow... what was I smoking back then.
Fixed in 58669fef and hyprwm/hyprland-protocols@6b3261e.
